### DISCUSSION TITLE
Lists of used tags

### DISCUSSION THREAD

**Discussion by snden** [vkbo/novelWriter#1127] [Ideas]
I think it would be useful if Novelwriter could also export, when building a Novel project, lists of used tags with links where they appear. Basically similar to the file preview window, where the option is at the bottom. List of references. In the resulting exported file, there could be, for example, below the own text:

Object1 ... Chapter One
                 Chapter Two
                 Chapter Four
Object2 ... Chapter One
                 Chapter Three
                 Chapter Four

Character1 ... Chapter One
                       Chapter Two
                       Chapter Three
                       Chapter Four
Character2 ... Chapter Two

also for locations

and so on, with links to Headings to the appropriate chapter. What do you think? It would help me to orient myself in which chapters the character (thing, ...) is in the story.
If it was possible already when writing the story, for example as part of the Outline preview, it would be excellent.

**Comment by vkbo** [comment]
If you open for instance a character note in the document viewer, and click on the references icon in the document footer, you will get a list of all documents that refer back to that note. That is for the time being the only place you can see where something is referenced.

There are already multiple feature suggestions discussing ways to present the information collected from tags and references. See for instance #76 and #854. I want to add a new view, similar to Outline, but for categories of tags. I still haven't figured out a solution, but I'm collecting ideas from users. Those two issues have a lot of ideas.
